selendroid-server/src/main/java/io/selendroid/server/model/DefaultAppiumUiAutomatorDriver.java

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright 2012-2014 eBay Software Foundation and selendroid committers.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.selendroid.server.model;
+
+import android.support.test.uiautomator.UiDevice;
+import android.test.InstrumentationTestCase;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.List;
+import java.util.Random;
+import java.util.Set;
+import java.util.UUID;
+
+import io.selendroid.server.ServerInstrumentation;
+import io.selendroid.server.android.InstrumentedKeySender;
+import io.selendroid.server.android.KeySender;
+import io.selendroid.server.android.internal.Dimension;
+import io.selendroid.server.common.exceptions.NoSuchMethodImplementationException;
+import io.selendroid.server.common.exceptions.SelendroidException;
+import io.selendroid.server.util.Preconditions;
+import io.selendroid.server.util.SelendroidLogger;
+
+
+public class DefaultAppiumUiAutomatorDriver implements SelendroidDriver {
```

Comment:
Can you change the name of this class? (This project is technically not 'appium', probably can just remove it and the name seems ok)